common/lib/user_input.py

```diff
@@ -35,6 +35,8 @@ class UserInput:
     OPTION_FILE = "file"  # file upload
     OPTION_HUE = "hue"  # colour hue
     OPTION_DATASOURCES = "datasources"  # data source toggling
+    OPTION_DATASOURCES_TABLE = "datasources_table" # a table with settings per data source
+    OPTION_DATASOURCES_TEXT = "datasources_text" # text input per data source (via dropdown)
```

Comment:
`OPTION_DATASOURCES_TEXT` seems to be unused

Reply:
Removed in [4a92b14](https://github.com/digitalmethodsinitiative/4cat/pull/428/commits/4a92b1488c324674d922b8836bdc968532399307)